src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -173,6 +178,11 @@ $pf-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl"
   &.pf-m-toggle-group {
     --pf-c-toolbar--spacer: var(--pf-c-toolbar__group--m-toggle-group--spacer);
 
+    &,
+    .pf-c-toolbar__group {
+      flex-wrap: nowrap;
```

Comment:
Is this necessary? Doesn't look like they're set to wrap to begin with.

Reply:
Good catch!